src/python/model/README.md

```diff
@@ -0,0 +1,68 @@
+# Qodana imitation model 
+
+## Architecture 
+`RobertaForSequenceClassification` model with `BCEWithLogitsLoss` solve multilabel classification task. 
```

Comment:
Could you please add some link to a description of RobertaForSequenceClassification and BCEWithLogitsLoss?